index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <title>My portfolio</title>
+  </head>
+  <body>
+    <section class="hero">
+      <header class="nav-container">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav class="nav-bar">
+          <ul class="menu-list">
+            <li class="menu-item"><a href="#">Portfolio</a></li>
+            <li class="menu-item"><a href="#">About</a></li>
+            <li class="menu-item"><a href="#">Contact</a></li>
+          </ul>
+          <div class="mob-menu">
+            <div class="mob-menu-bar"></div>
+            <div class="mob-menu-bar"></div>
+            <div class="mob-menu-bar"></div>
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.), Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**Original Design**
![Screenshot from 2023-04-29 00-30-35](https://user-images.githubusercontent.com/97834160/235264647-e67a6884-d0bd-434b-8f85-62807311e340.png)

**Your Design**
![Screenshot from 2023-04-29 00-30-09](https://user-images.githubusercontent.com/97834160/235264750-b0454275-fd17-48c1-b2e1-c236800e0d15.png)